frontend/src/scenes/experiments/ExperimentMetricConversionWindowFilter.tsx

```diff
@@ -0,0 +1,98 @@
+import { LemonInput } from 'lib/lemon-ui/LemonInput'
+import { LemonLabel } from 'lib/lemon-ui/LemonLabel'
+import { LemonRadio } from 'lib/lemon-ui/LemonRadio'
+import { LemonSelect, LemonSelectOption } from 'lib/lemon-ui/LemonSelect'
+import { capitalizeFirstLetter, pluralize } from 'lib/utils'
+import { TIME_INTERVAL_BOUNDS } from 'scenes/insights/views/Funnels/FunnelConversionWindowFilter'
+
+import { ExperimentMetric } from '~/queries/schema/schema-general'
+import { FunnelConversionWindowTimeUnit } from '~/types'
+
+export function ExperimentMetricConversionWindowFilter({
+    metric,
+    handleSetMetric,
+}: {
+    metric: ExperimentMetric
+    handleSetMetric: (newMetric: ExperimentMetric) => void
+}): JSX.Element {
+    const options: LemonSelectOption<FunnelConversionWindowTimeUnit>[] = Object.keys(TIME_INTERVAL_BOUNDS).map(
+        (unit) => ({
+            label: capitalizeFirstLetter(pluralize(metric.conversion_window ?? 72, unit, `${unit}s`, false)),
+            value: unit as FunnelConversionWindowTimeUnit,
+        })
+    )
+    const intervalBounds = TIME_INTERVAL_BOUNDS[metric.conversion_window_unit ?? FunnelConversionWindowTimeUnit.Day]
```

Comment:
⛏️ Complete and absolute nitpick:  you could extract these as functions that take an `ExperimentMetric` argument. Then, they could also be moved out into `FunnelConversionWindowFilter` scene, removing the `TIME_INTERVAL_BOUNDS` import (hiding that away).

Reply:
As the functions are only used here and are pretty straight forward, I think I prefer leaving them there for now.